client/src/utils/contants.js

```diff
@@ -2,3 +2,4 @@
 
 export const JOB_TYPES = ["Full Time", "Part Time", "Casual", "On-Call"];
 export const STATUS_TYPES = ["Active", "R&I", "Closed"];
+export const COMPENSATION_RATES = ['Hourly', 'Weekly', 'Annually', 'Commision', 'Base & Commission']
```

Comment:
"Base & Commission" might need to be changed to "Base and Commission" since MySQL doesn't allow ampersands in ENUM values.

Reply:
```suggestion
export const COMPENSATION_RATES = ['Hourly', 'Weekly', 'Annually', 'Commission', 'Base and Commission']
```